python/cudf/cudf/core/udf/_ops.py

```diff
@@ -53,3 +60,7 @@
     operator.gt,
     operator.ge,
 ]
+
+
+# currently only numeric types are supported.
+SUPPORTED_TYPES = NUMERIC_TYPES | BOOL_TYPES | DATETIME_TYPES | TIMEDELTA_TYPES
```

Comment:
I didn't find the previous comment I was seeking. To elaborate a bit more, it feels a bit awkward to define this here because (1) it is only imported/used in one other file and (2) the types don't really seem to fit in a file named `_ops.py` because they aren't defining operations. Finally, the name could be clarified as `JIT_SUPPORTED_TYPES`, similar to the change of the function name `_is_supported_type` to `_is_jit_supported_type`. Not a blocker but might be nice to consider.